doc/classes/OS.xml

```diff
@@ -611,6 +611,25 @@
 				The first element holds the driver name, such as [code]nvidia[/code], [code]amdgpu[/code], etc.
 				The second element holds the driver version. For example, on the [code]nvidia[/code] driver on a Linux/BSD platform, the version is in the format [code]510.85.02[/code]. For Windows, the driver's format is [code]31.0.15.1659[/code].
 				[b]Note:[/b] This method is only supported on Linux/BSD and Windows when not running in headless mode. On other platforms, it returns an empty array.
+				[b]Note:[/b] This method will run slowly the first time it is called in a session; it can take several seconds depending on the operating system and hardware. It is blocking if called on the main thread, so it's recommended to call it on a separate thread using [Thread]. This allows the engine to keep running while the information is being retrieved:
```

Comment:
Windows implementation has 5-sec timeout on one of the internal calls.

Since calling it on thread is now the recommended way to use it (which I agree with), it might be worth mentioning that the function is not thread safe, and should not be called from multiple threads at the same time.